tencentcloud/resource_tc_kubernetes_cluster.go

```diff
@@ -231,6 +231,27 @@ func TkeCvmCreateInfo() map[string]*schema.Schema {
 				return
 			},
 		},
+		// payment
+		"instance_charge_type": {
+			Type:         schema.TypeString,
+			Optional:     true,
+			ForceNew:     true,
+			Default:      CVM_CHARGE_TYPE_POSTPAID,
+			ValidateFunc: validateAllowedStringValue(TKE_WORKER_CHARGE_TYPE),
+			Description:  "The charge type of instance. Valid values are `PREPAID` and `POSTPAID_BY_HOUR`, The default is `POSTPAID_BY_HOUR`.",
```

Comment:
should doc "the international account only support  PREPAID"

Reply:
Will added, and please check the same comments in `resource_tc_instance`.